### ISSUE TITLE
Default template should not appear on Public templates page

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#1426]
The default template should not appear on the list of public funder templates. The default template receives a status of publicly visible so that it is available in both the customizable templates list and is available when creating a plan.

We need to filter out `template.is_default?` from the list of public templates

**Comment by briri** [comment]
This is no longer an issue in the newly revised `template.customize!` method. The visibility is always set to organisational when customizing.

**Comment by stephaniesimms** [comment]
I think this is fixed but there might be some bad data from prior testing. @briri can you check that the DCC default template only belongs to DCC? I see a second customized version that belongs to UCOP. we should only have one, uncustomized version (either org can be the owner actually depending on our config for DMPTool default template and guidance)

**Comment by briri** [comment]
yep. As you suspected they were both marked as default due to an old bug. I just removed the default flag from the UCOP one and then customized it again to ensure that it doesn't appear in the list.
